devel/gettext/Portfile

```diff
@@ -6,9 +6,7 @@ PortGroup               muniversal 1.0
 PortGroup               clang_dependency 1.0
 
 name                    gettext
-epoch                   2
-version                 0.19.8.1
-revision                2
```

Comment:
There should be a `revision 0` line in each subport and also in the `if {${subport} eq ${name}}` section.

Reply:
I added a revision 0 in each subport and in the `if {${subport} eq ${name}}` section.